plugins/api-docs/src/alpha.tsx

```diff
@@ -0,0 +1,205 @@
+  name: 'api-definition',
+  loader: () =>
+    import('./components/ApiDefinitionCard').then(m =>
+      compatWrapper(<m.ApiDefinitionCard />),
+    ),
+});
+
+const ApiDocsConsumedApisEntityCard = createEntityCardExtension({
+  name: 'consumed-apis',
+  // Ommiting configSchema for now
+  // we are skipping variants, see: https://github.com/backstage/backstage/pull/22619#discussion_r1477333252
+  // and columns are too complex to map to zod
+  loader: () =>
+    import('./components/ApisCards').then(m =>
+      compatWrapper(<m.ConsumedApisCard />),
+    ),
+});
+
+const ApiDocsProvidedApisEntityCard = createEntityCardExtension({
+  name: 'provided-apis',
+  // we are skipping variants, see: https://github.com/backstage/backstage/pull/22619#discussion_r1477333252
+  // and columns are too complex to map to zod
+  loader: () =>
+    import('./components/ApisCards').then(m =>
+      compatWrapper(<m.ProvidedApisCard />),
+    ),
+});
+
+const ApiDocsConsumingComponentsEntityCard = createEntityCardExtension({
+  name: 'consuming-components',
+  // Ommiting configSchema for now
+  // we are skipping variants, see: https://github.com/backstage/backstage/pull/22619#discussion_r1477333252
+  loader: () =>
+    import('./components/ComponentsCards').then(m =>
+      compatWrapper(<m.ConsumingComponentsCard />),
+    ),
+});
+
+const ApiDocsProvidingComponentsEntityCard = createEntityCardExtension({
+  name: 'providing-components',
+  // Ommiting configSchema for now
+  // we are skipping variants, see: https://github.com/backstage/backstage/pull/22619#discussion_r1477333252
+  loader: () =>
+    import('./components/ComponentsCards').then(m =>
+      compatWrapper(<m.ProvidingComponentsCard />),
+    ),
+});
+
+const ApiDocsDefinitionEntityContent = createEntityContentExtension({
+  name: 'definition',
+  defaultPath: '/defintion',
+  defaultTitle: 'Definition',
+  filter: 'kind:api',
+  loader: async () =>
+    import('./components/ApiDefinitionCard').then(m =>
+      compatWrapper(
+        <Grid container spacing={3}>
+          <Grid item xs={12}>
+            <m.ApiDefinitionCard />
+          </Grid>
+        </Grid>,
+      ),
+    ),
+});
+
+const ApiDocsApisEntityContent = createEntityContentExtension({
+  name: 'apis',
+  defaultPath: '/apis',
+  defaultTitle: 'APIs',
+  filter: 'kind:component',
+  loader: async () =>
+    import('./components/ApisCards').then(m =>
+      compatWrapper(
+        <Grid container spacing={3} alignItems="stretch">
+          <Grid item xs={12}>
+            <m.ProvidedApisCard />
+          </Grid>
+          <Grid item xs={12}>
+            <m.ConsumedApisCard />
+          </Grid>
+        </Grid>,
+      ),
+    ),
+});
+
+export default createPlugin({
+  id: 'api-docs',
+  routes: {
+    root: convertLegacyRouteRef(rootRoute),
+  },
+  externalRoutes: {
+    registerApi: convertLegacyRouteRef(registerComponentRouteRef),
+  },
+  extensions: [
```

Comment:
These don't follow the naming patterns at https://backstage.io/docs/frontend-system/architecture/naming-patterns#extensions. All internal so not a huge deal, but nice to not have things drift too much

Reply:
Fixed 🙌🏻